projects/map-ol/src/lib/map-ol.service.ts

```diff
@@ -1055,6 +1201,11 @@ export class MapOlService {
       layeroptions.extent = transformExtent(l.bbox.slice(0, 4) as [number, number, number, number], WGS84, this.getProjection().getCode());
     }
 
+    if (l.subdomains) {
```

Comment:
This was the Idee of the edge case if there could be multiple WFS layers with the same Data, the they could fetch it with subdomains.
But I don't really know it this makes sense in case of WFS.